### ISSUE TITLE
Debugger command missing vitest command

### ISSUE THREAD

**Issue by johtso** [vitest-dev/vscode#140]
I've managed to get test running working in my monorepo using workspaces.

When I try and run a debug, the command that gets executed seems to be missing the vite command:
```
/Users/human/.nvm/versions/node/v18.0.0/bin/node /Users/human/Code/remix-cloudflare-experiment/packages/remix-app/tests/encryption.test.ts -t should encode and decode --api.port 55438 --api.host 127.0.0.1
```

This means I get an error because of node trying to execute a typescript file.

Any ideas why this might be happening? There must at least be some error checking missing that's allowing a command to execute with no vitest command.

Looking at the code it seems like it usually tries to fall back to a default though..
https://github.com/vitest-dev/vscode/blob/bf715411a0365d1a530f2cc2a30c0b611177392d/src/pure/runner.ts#L67-L69

My settings.json:
```json
{
    "vitest.enable": true,
    "vitest.commandLine": "pnpm test",
    "vitest.nodeEnv": {
        "NODE_OPTIONS": "--experimental-global-webcrypto"
    },
}
```

the test command in my package:
```ts
"test": "NODE_OPTIONS='--experimental-global-webcrypto' vitest run",
```

**Comment by koistya** [comment]
A couple more observations:

- `"vitest.commandLine": "yarn vitest"` doesn't work for debugging
- `"vitest.commandLine": "yarn vitest --"` doesn't work for running tests (while without `--` does)

**Comment by segevfiner** [comment]
Also hit this. Had to run vitest manually in the "Javascript Debug Terminal" to debug, but this kinda defeats the point of the extension.

**Comment by coryvirok** [comment]
Any update on this? Having to create a new Javascript Debug Terminal each time I want to debug a test is kind of painful.

**Comment by sheremet-va** [comment]
This should be fixed in pre-release 0.5.0 and higher. Note that the extension now requires Vitest 1.4.0 or higher.

We don't use Vitest CLI anymore, but call the API directly, so this is no longer a problem.
